paddle/fluid/pybind/imperative.cc

```diff
@@ -774,8 +800,10 @@ void BindImperative(py::module *m_ptr) {
                  imperative::AllReduce(self.Var(), self.MutableVar(), strategy);
                } else {
                  PADDLE_THROW(platform::errors::Unimplemented(
-                     "Imperative SelectedRows allreduce is not supported when "
-                     "paddle is compiled with NCCL verison lower than v2.2.12. "
+                     "Imperative SelectedRows allreduce is not supported "
+                     "when "
```

Comment:
Why break one word? Can you make the line pretty?

Reply:
Sorry, fixed it.